arcgis-ios-sdk-samples/Cloud and portal/Token authentication/TokenAuthenticationViewController.swift

```diff
@@ -34,6 +34,12 @@ class TokenAuthenticationViewController: UIViewController {
         return AGSMap(item: portalItem)
     }
     
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        
+        AGSAuthenticationManager.shared().credentialCache.removeAllCredentials()
```

Comment:
The credentials are cleared at the end of this [sample](https://github.com/Esri/arcgis-runtime-samples-ios/blob/8e269567c08f19c626a62940b60dd4003ee9d651/arcgis-ios-sdk-samples/Cloud%20and%20portal/Authenticate%20with%20OAuth/AuthenticateWithOAuthViewController.swift#L61-L64) as well. So it is redundant but we should clear the cache in `Authenticate with OAuth` to showcase the correct workflow.